### PR TITLE
fix: py3-apache-beam + update to 2.53.0

### LINKED ISSUE #8792: wolfi-bot/py3-apache-beam
failed to bump package py3-apache-beam to version 2.52.0: got 404 Not Found when fetching https://files.pythonhosted.org/packages/89/d5/48ca435cac3d73a422bcb1f51ffdeebbcee6b89107ebc5b0c3cc2f35857a/apache-beam-2.52.0.zip

### PR DESCRIPTION
**Pull request by ajayk**
Fixes https://github.com/wolfi-dev/os/issues/8792

### CODE REVIEW

**Comment by rawlingsj** [comment]
error:

```
⚠️  x86_64    |   File "/tmp/pip-build-env-q3qdmm7n/overlay/lib/python3.12/site-packages/setuptools/__init__.py", line 10, in <module>
⚠️  x86_64    |     import distutils.core
⚠️  x86_64    | ModuleNotFoundError: No module named 'distutils'
```

**Comment by rawlingsj** [comment]
Requires py3-setuptools

File: envoy/luajit.patch
Comment (rawlingsj) [inline envoy/luajit.patch:1]: This was accidentally removed?

Comment (ajayk) [reply envoy/luajit.patch:1]: seems like it , let me revert this change

**Review by dlorenc — APPROVED** [review]

**Pull request merged**